buildroot/package/systemd/systemd.mk

```diff
@@ -41,7 +41,8 @@ SYSTEMD_CONF_OPTS += \
 	-Dumount-path=/usr/bin/umount \
 	-Dnobody-group=nogroup \
 	-Didn=true \
-	-Dnss-systemd=true
+	-Dnss-systemd=true \
+	-Dapparmor=false
```

Comment:
I'm not sure what the policy is regarding changes to the vendored buildroot.

Looks like upstream has already switched to systemd v246.5, and also has new flags to enable/disable apparmor:

https://github.com/buildroot/buildroot/blob/master/package/systemd/systemd.mk#L56-L61

Reply:
I see, in that case just add a patch to `buildroot-patches/` as well, kinda as a marker to check when we bump buildroot.

Reply:
Yeah buildroot upstream support now apparmor